architectural-design-records/dependency-management-automation.md

```diff
@@ -0,0 +1,42 @@
+## Summary
+
+Use a Renovate bot to automatically update dependencies in our GitHub repository.
+
+### Problem
+
+Keeping dependencies up to date can be a time-consuming task, especially for large projects with many dependencies. It can be easy to miss updates, and updating dependencies manually can introduce errors and inconsistencies. As the number of charts that the project supports will grow,
+keeping up-to-date with the latest releases manually is a time consuming effort.
+
+### Decision
+
+We will use a Renovate bot to automatically update dependencies in our GitHub repository. The bot will check for updates on a regular basis and open pull requests for any updates that are found and if merge automatically if the checks will pass and change is backward compatible (patch or minor). The pull requests with major changes, will then be reviewed and merged by the project maintainers.
```

Comment:
- `on a regular basis` -> `regularly`
- `...with major changes, will then be reviewed...` -> comma not needed
- `...if the checks will pass and change is backward compatible...` -> `if the checks will pass and the change is backward compatible`
- `...and if merge automatically...` -> `and merge automatically`